src/03-modality-agnostic-files.md

```diff
@@ -98,20 +98,34 @@ sub-control02 12  F control
 sub-patient01 33  F patient
 ```
 
+## Phenotype files (questionnaires, etc.)
```

Comment:
Could you say why? I was including this because many people might want to include "questionnaire-like" data but don't know what to make of "phenotypic data". It was intended as an improvement of findability.

Reply:
Aesthetics. It's still findable by a text search, which is really the only sensible way to approach the document, but it won't clutter the TOC with a kind of informal parenthetical.

If we want a title that will stick out as probably containing questionnaire results and other assessments, what about `Phenotypic and Assessment Data`?

Reply:
sounds good, thanks 👍

Reply:
@sappelhoff Updated the suggestion.